src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -26,8 +30,10 @@
 @State(Scope.Thread)
 public class Benchmarks {
 
-    String bibtexString;
-    BibDatabase database = new BibDatabase();
+    private String bibtexString;
+    private final BibDatabase database = new BibDatabase();
+    private final List<String> latexConversionStrings = new ArrayList<>();
```

Comment:
Except that each run converts 1000 strings. But if a static string should
be used I guess it is enough to create a single (quite long) string.